### PR TITLE
disambiguation cli: cast to string record uuid

### PR DESCRIPTION
**Pull request by MJedr**

### CODE REVIEW

File: backend/inspirehep/disambiguation/cli.py [hunk 1]
```diff
@@ -108,7 +108,7 @@ def _get_all_not_disambiguated_records_search():
 
 def _send_celery_group_disambiguation_task(uuids, batch_size):
     records = LiteratureRecord.get_records(uuids)
-    input_data = ((record.id, record.model.version_id, True) for record in records)
+    input_data = ((str(record.id), record.model.version_id, True) for record in records)
```
Comment (drjova) [inline backend/inspirehep/disambiguation/cli.py:111]: could we add a test for that?

Comment (MJedr) [reply backend/inspirehep/disambiguation/cli.py:111]: what exactly do you want to test in casting from uuid type to string?

Comment (drjova) [reply backend/inspirehep/disambiguation/cli.py:111]: yes it's a cast, but we should have caught this during tests if it's causing errors. WDYT?

Comment (MJedr) [reply backend/inspirehep/disambiguation/cli.py:111]: yes, I agree, the problem is that we have a test covering this case and it's working correctly, so I don't know what else can I add

**Pull request merged**